### ISSUE TITLE
Solidus 2.8 Migration for Promotions Throws Exception

### ISSUE THREAD

**Issue by brchristian** [solidusio/solidus#3119]
I’m trying to update our app from Solidus 2.7.1 to Solidus 2.8.1. The `remove_code_from_spree_promotions` migration fails with an exception.

**Solidus Version**:
2.8.1
(Running on Rails 5.1.6.1)

**Current behavior**
I ran `rake railties:install:migrations` to retrieve [`remove_code_from_spree_promotions.rb`](https://github.com/solidusio/solidus/blob/master/core/db/migrate/20190106184413_remove_code_from_spree_promotions.rb).

I changed the handler from `Solidus::Migrations::PromotionWithCodeHandlers::RaiseException` to [`Solidus::Migrations::PromotionWithCodeHandlers::MoveToSpreePromotionCode`](https://github.com/solidusio/solidus/blob/master/core/db/migrate/20190106184413_remove_code_from_spree_promotions.rb#L39)

I ran `rake db:migrate`, and got an exception as follows:

```
ActiveRecord::SubclassNotFound: Invalid single-table inheritance type: Spree::Promotion is not a subclass of RemoveCodeFromSpreePromotions::Promotion
```

Stack trace was 

[gems/solidus_core-2.8.1/lib/solidus/migrations/promotions_with_code_handlers.rb:39:in 'call'](https://github.com/solidusio/solidus/blob/master/core/lib/solidus/migrations/promotions_with_code_handlers.rb#L39)
[db/migrate/20190222024608_remove_code_from_spree_promotions.spree.rb:18:in 'up'](https://github.com/solidusio/solidus/blob/master/core/db/migrate/20190106184413_remove_code_from_spree_promotions.rb#L18)

**Expected behavior**
Migration should execute correctly.

**Desktop (please complete the following information):**
 - OS: MacOS Mojave

**Comment by seand7565** [comment]
Looks like this was fixed with #3108 - though the fix is only on master ATM.

**Comment by brchristian** [comment]
Closing this as I think #3108 has all the relevant info and discussion. Thanks for the heads up!
